src/main/java/org/broadinstitute/barclay/help/DefaultDocWorkUnitHandler.java

```diff
@@ -243,7 +243,7 @@ protected void addHighLevelBindings(final DocWorkUnit workUnit)
      */
     protected void addCustomBindings(final DocWorkUnit currentWorkUnit) {
         final String tagFilterPrefix = getTagPrefix();
-        Arrays.stream(currentWorkUnit.getClassDoc().inlineTags())
+        Arrays.stream(currentWorkUnit.getClassDoc().tags())
```

Comment:
Also, I realized from looking into this more, and from looking at the GATK beta doc output, that there is a bug in Barclay docgen where in some cases we don't include javadoc after the first embedded tag, at least in some cases. I'll try to fix that ASAP since its fairly high priority. In the meantime, lets leave this PR open for the short term so we can retain this discussion thread, even though we'll probably replace it with another PR with a more comprehensive fix.